components/AnchorCollapse/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@gemeente-denhaag/anchor-collapse",
+  "version": "0.1.0",
+  "description": "An anchor collapse component",
+  "bugs": "https://github.com/nl-design-system/denhaag/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nl-design-system/denhaag.git",
+    "directory": "components/AnchorNavigation"
```

Comment:
I think this should be `AnchorCollapse`

Reply:
Sharp! Missed that one. Will update right away.